drillsrs/cmd/study.py

```diff
@@ -21,10 +22,7 @@ def _learn_single_card(
     if mode is Mode.reversed or mode is Mode.mixed and random.random() > 0.5:
         raw_question, raw_answers = random.choice(raw_answers), [raw_question]
 
-    question = 'Question: %s' % raw_question
-    if card.tags:
-        question += ' [%s]' % util.format_card_tags(card.tags)
-    util.ask(question)
+    print_question(raw_question, raw_answers, card.tags)
```

Comment:
This change causes the script to show answers immediately during the study session, which is not desired.

Reply:
Sorry! Tried to fix it :-v